---
permalink: /software/
title: "Software"
author_profile: true
redirect_from: 
  - /md/
  - /software.html
---

Here is a list of open source software that I developed or had major contribution to. 
These tools are typically implementation of machine learning models originated from research projects.
For the research topics I'm interested in, please check the [Research](https://s6juncheng.github.io/research/) site. 

### [MMSplice & MTSplice](https://github.com/gagneurlab/MMSplice_MTSplice)

Predict variant effect on splicing. MMSplice is the winning model of the [CAGI5 splicing challenge](https://genomeinterpretation.org/content/vex-seq). 
MMSplice is also integrated in the popular general purpose variant effect predictor [CADD](https://cadd.gs.washington.edu/news).
MTSplice enhances MMSplice by predicting tissue-specific variant effect.
Currently, Muhammed Hasan Çeli and I are maintaining the tool.

### [ggpval](s6juncheng.github.io/ggpval)

A R package to add statistical test and *P*-value annotations to ``ggpplot``. 
Currently, the user community and myself are maintaining the tool. 

### [DCC](https://github.com/dieterich-lab/DCC)

A python package to detect circRNAs from next-generation sequence data.
Currently, the Dieterich lab is maintaining the tool.

--------

I contributed the following projects:

### [kipoi](kipoi.org)
Kipoi (pronounce: kípi; from the Greek κήποι: gardens) is an API and a repository of ready-to-use trained models for genomics. 
It currently contains 2133 different models, covering canonical predictive tasks in transcriptional and post-transcriptional gene regulation. 
